api/docs/v2/storefront/index.yaml

```diff
@@ -5078,7 +5109,7 @@ components:
         included:
           type: array
           items:
-            oneOf:
+            anyOf:
```

Comment:
@MatthewKennedy this change breaks displaying of States in includes

<img width="1127" alt="Screenshot 2021-08-17 at 17 15 11" src="https://user-images.githubusercontent.com/55154/129752977-c7674a8e-c37b-4930-830c-f142c0561e46.png">

Reply:
I'll look into it...

Reply:
@damianlegawiec is this PR code live on: https://api.spreecommerce.org/docs/api-v2/b3A6MzE0Mjc2Nw-get-a-country

Reply:
I can't see the states in the live docs either:

**Live Docs:**
<img width="927" alt="Screenshot 2021-08-17 at 18 34 08" src="https://user-images.githubusercontent.com/1240766/129773889-ea047a37-ead8-4a9c-a2e8-e7bff1ee7ee3.png">

**New Version**
<img width="973" alt="Screenshot 2021-08-17 at 18 35 34" src="https://user-images.githubusercontent.com/1240766/129774011-0e3d4db3-b13b-4607-862c-853351915efc.png">